.travis.yml

```diff
@@ -3,7 +3,7 @@ language: go
 go_import_path: go.uber.org/yarpc
 cache:
   directories:
-    - $HOME/.yarpc-go
+    - $HOME/.cache/yarpc-go
```

Comment:
See comment about `CACHE_BASE`, but are you sure added environment variables are available here? Ie verify this on Travis CI.